h2o-py/tests/testdir_algos/automl/target_encoding/pyunit_target_encoding_model.py

```diff
@@ -36,10 +36,13 @@ def test_target_encoding_fit_method():
     assert transformed.nrows == 1309
     
     # Test fold_column proper handling + kfold data leakage strategy defined
-    te = H2OTargetencoderEstimator(encoded_columns = teColumns, target_column = targetColumnName, fold_column = "pclass",
-                                   k = 0.7, f = 0.3, data_leakage_handling = "kfold")
-    te.train(training_frame = trainingFrame)
+    te = H2OTargetencoderEstimator(k=0.7, f=0.3)
+    te.train(training_frame=trainingFrame, fold_column="pclass", data_leakage_handling="kfold",
```

Comment:
I think it's debatable whether `data_leakage_handling` belongs in the constructor or the train method... but I think we just put arguments that relate to the given `training_frame` in `.train()`.  So my instinct is to keep `data_leakage_handling` in the constructor since it's a characteristic of the TE process, rather than the frame you are operating on.

Reply:
Removed.